networks/duality-devnet/mnemonics.txt

```diff
@@ -0,0 +1,4 @@
+reveal gloom baby scan escape shine flock kind marriage sad clinic affair invite dove document youth weekend brave mutual chief sunny return myself sorry
```

Comment:
since this is needed locally to create the resources I guess this is ok. 

but if we put the docker-compose resource creation step in CI then we could move the mnemonics to the CI environment variables right?

Reply:
I've switched them to environment variables and moved them docker-compose.override.yml for clarity/simplicity. But I'm inclined to keep them in the repo so there is an easy record of what they are. Otherwise, it's kinda PITA to hunt them down later. I suppose there's a risk that someone could find the devnet and do something annoying, but I don't really see a big problem.

Reply:
I like this setup 👍 its both explicit about the mnemonics used but its also very clear that the mnemonics are optional and we could override them in any build by passing other environment variables.